src/base/Module.zig

```diff
@@ -30,6 +30,7 @@ pub const Idx = List.Idx;
 /// module itself and builtin modules.
 pub const Store = struct {
     modules: List,
+    ident_store: Ident.Store,
```

Comment:
We should be using a single ident store between this and `ModuleEnv`. Maybe this can be a reference to `ModuleEnv`'s ident store and we can pass it in to `init`?